arches/app/media/css/arches.scss

```diff
@@ -6242,6 +6242,199 @@ th.notif-type {
     padding-right: 20px;
 }
 
+@media only screen and (min-width : 768px) {
+    .tabbed-workflow-title-bar {
+        .workflow-name-container {
+            display: flex;
+            flex-direction: row;
+        }
+
+        .workflow-top-control {
+            display: inline-flex;
+            max-width: 335px;
+            justify-content: flex-end;
+        }
+    }
+
+    .step-metadata {
+        display: flex;
+    }
+
+    .workflow-step-title,
+    .workflow-plugin .btn .succinct,
+    .ep-tools .fa-sign-out {
+        display: none
+    }
+}
```

Comment:
Can this work outside of a screen break?

Reply:
yeah; I removed the break.  In this case, it was actually yanking these styles below the screen break.  This was helpful, but not really necessary.